js/ui/newCard.js

```diff
@@ -0,0 +1,63 @@
+import { getCardList } from './cardList.js';
+import { addCard } from '../data/actions.js';
+import { generateCounter } from './cardCounter.js';
+import { createState } from '../data/createState.js';
+
+let appState = createState();
```

Comment:
**issue:** You should not initialize `appState` twice - it should be done once somewhere in your `app.js` component. You `UI` files should not know anything about `data` layer